github/ci/prow-deploy/files/jobs/kubernetes-sigs/cluster-api-provider-kubevirt/cluster-api-provider-kubevirt-presubmits.yaml

```diff
@@ -0,0 +1,33 @@
+presubmits:
+  kubernetes-sigs/cluster-api-provider-kubevirt:
+  - always_run: true
+    optional: true
+    decorate: true
+    skip_report: true
+    cluster: prow-workloads
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 7h0m0s
```

Comment:
ha, 7 hours. doubt we'd hit that. if we ever hang for some reason, it might be nice ot have this slightly more realistic so we release resources quicker.